components/becomeSponsorButton.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div id="sponzaButtonContainer">
+    <img
+      src="~/assets/graphics/leafLeft.svg"
+    >
+    <a id="sponzaButton" href="mailto:logistics@nwplus.io?subject=CMD-F Sponsorship">
```

Comment:
If they find out about this you're going to get roasted

Reply:
oop